pkg/csource/options.go

```diff
@@ -8,10 +8,91 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"sort"
+	"strings"
 
 	"github.com/google/syzkaller/pkg/mgrconfig"
 )
 
+type Feature struct {
+	Name        string
+	Description string
+	Enabled     bool
+}
+
+type Features map[string]Feature
+
+func defaultFeatures(value bool) Features {
+	return map[string]Feature{
+		"tun":         {"tun", "setup and use /dev/tun for packet injection", value},
+		"net_dev":     {"net_dev", "setup more network devices for testing", value},
+		"net_reset":   {"net_reset", "reset network namespace between programs", value},
+		"cgroups":     {"cgroups", "setup cgroups for testing", value},
+		"binfmt_misc": {"binfmt_mist", "setup binfmt_misc for testing", value},
+	}
+}
+
+func enableAllFeatures(features Features) Features {
```

Comment:
Move this and disableAllFeatures after ParseFeaturesFlags. This is not C, you don't have to show people the least important stuff first ;)

Reply:
Done.